src/components/Notifications/Notifications.jsx

```diff
@@ -0,0 +1,20 @@
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import styles from './Notifications.module.css'
+
+export const Notifications = ({message, icon, className, color}) => {
```

Comment:
навізо ти створив `variant` в `props` і навіть не використовуєш його?

```suggestion
export const Notifications = ({variant, className, children}) => {
```